test/EIP20CoGateway/_redeemRawTx.js

```diff
@@ -0,0 +1,212 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const chai = require('chai');
+const Web3 = require('web3');
+const sinon = require('sinon');
+
+const assert = chai.assert;
```

Comment:
```suggestion
const { assert } = chai;
```
Maybe? ESLint in mosaic contracts would complain.